com.github.skullernet.q2pro.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: com.github.skullernet.q2pro
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: q2pro.sh
+rename-icon: q2pro
+rename-desktop-file: q2pro.desktop
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+modules:
+  - name: q2pro
+    no-autogen: true
+    no-make-install: true
+    make-args:
+      - CONFIG_PNG=y
+      - PNG_CFLAGS=
+      - PNG_LIBS=-lpng16
+      - CONFIG_JPEG=y
+      - CONFIG_HTTP=y
+      - CONFIG_OPENAL=y
+      - CONFIG_DIRECT_SOUND=y
+      - CONFIG_PATH_DATA=/app/bin
+      - CONFIG_PATH_LIB=/app/bin
+      - CONFIG_PATH_HOME=/var/data
+    build-commands:
+      - install -D q2pro.sh -t /app/bin
+      - desktop-file-edit --set-key=Exec --set-value=q2pro.sh src/unix/res/q2pro.desktop
+      - install -D q2pro -t /app/bin
+      - install -D src/client/ui/q2pro.menu -t /app/bin/baseq2
+      - install -D game$(uname -m).so -t /app/bin/baseq2
+      - install -Dm644 com.github.skullernet.q2pro.appdata.xml -t /app/share/metainfo
+      - install -Dm644 src/unix/res/q2pro.desktop -t /app/share/applications
+      - install -Dm644 q2pro.svg -t /app/share/icons/hicolor/scalable/apps
+    sources:
+      - type: git
+        url: https://github.com/skullernet/q2pro.git
+        tag: r1826
+        commit: 7ad132affeba68a91984852851630b2fe417c307
+      - type: file
+        path: com.github.skullernet.q2pro.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html